css/index.css

```diff
@@ -1,11 +1,95 @@
-/* Example of @import use
-This imports the `reset.css` file */
-@import url('./reset.css');
-
-/* For testing purposes, delete this code */
-body {
-  font-size: max(20vw, 400px);
-  width: min(20vw, 400px);
-  margin: 20px;
+/* stylelint-disable font-family-name-quotes */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:wght@100;300;400;500;700;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:ital,wght@0,100;0,200;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:wght@100;300;400;500;700;900&display=swap');
+
+* {
+  margin: 0;
   padding: 0;
 }
```

Comment:
Let's add the general font to this, then we can specify the font in other places when it's defferent